src/main/java/com/solvd/utilities/ConnectionPool.java

```diff
@@ -0,0 +1,85 @@
+package com.solvd.utilities;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.sql.Connection;
+
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import java.util.concurrent.ArrayBlockingQueue;
+
+public class ConnectionPool {
+    private ArrayBlockingQueue<Connection> pool;
+    private ArrayBlockingQueue<Connection> inUseConnections;
+    private int maxConnections;
+    private static String url;
+    private static String username;
+    private static String password;
+    private static ConnectionPool connectionPool;
+
+
+    //load DB properties
+    static {
+        try {
+            FileReader fileReader = new FileReader(".\\src\\main\\resources\\db.properties");
+            Properties properties = new Properties();
+            properties.load(fileReader);
+            url = properties.getProperty("url");
+            username = properties.getProperty("user");
+            password = properties.getProperty("password");
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
use try-with-resources statement because FileReader implements the  AutoCloseable interface